packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -6,6 +6,10 @@ import { ISemanticTextColors } from './ISemanticTextColors';
  * The collection of all semantic slots for colors used in themes.
  *
  * Note: text colors are defined in ISemanticTextColors.ts.
+ * We are aware that the usage of semantic slots vs. palette slots is not consistent across the Fabric controls code.
```

Comment:
I wouldn't say it's not consistent, that implies that slot usage is incorrect. Rather, I'd say something about how we still use both, whereas the desired state (I hope) is to move completely to semantic slots. (We should also state our desired end goal.)

Reply:
Ok addressed this and the above comments with a rewrite!